qiskit/__init__.py

```diff
@@ -127,3 +126,23 @@ def __getattr__(self, attr):
 
 Aer = AerWrapper()
 IBMQ = IBMQWrapper()
+
+__all__ = [
+    "Aer",
+    "AncillaRegister",
+    "BasicAer",
+    "ClassicalRegister",
+    "_config",
```

Comment:
I added because `_config = _user_config.get_config()` and I assumed that intentional. However, checking in the history it seems that was added by #3362 and ended up being leftovers. So I removed all the related lines in 76370cc92d83a58813cc17ded2055167feb1e741

Reply:
It's still usable, but at the very least I'd think something like that ought to be through qualified access, so we're fine not to include it in `__all__`.